backend/Dockerfile

```diff
@@ -52,17 +50,20 @@ RUN pip install --no-cache psycopg[binary] gunicorn
 # copy project
 COPY --chown=app:app . $APP_HOME
 
-# change to the app user
-USER app
+# # change to the app user
+# USER app
 
 ARG SECRET_KEY
 ENV SECRET_KEY=${SECRET_KEY}
 ENV DJANGO_SETTINGS_MODULE=backend.production_settings
 
+# RUN python manage.py collectstatic --no-input --clear
+# RUN sleep 3
+
 # run entrypoint.prod.sh
-EXPOSE 8000
+EXPOSE 80
 ENTRYPOINT ["/bin/sh", "entrypoint.prod.sh"]
-RUN echo -e 'gunicorn backend.wsgi:application --bind 0.0.0.0:8000\n' > entrypoint.prod.sh
+RUN echo -e 'gunicorn backend.wsgi:application --bind 0.0.0.0:8000 --workers=3\n' > entrypoint.prod.sh
```

Comment:
Isn't 3 workers arbitrary?
Maybe you should leave it as default, or else make it configurable with some variable?

Reply:
From the docs for that property:
```
A positive integer generally in the 2-4 x $(NUM_CORES) range. You’ll want to vary this a bit to find the best for your particular application’s work load.
```
On a pod the number of cores is 1, so we're in the 2-4 range, so 3 is the average. (The default is 1, so we do actually want to overwrite it). We could certainly make it a variable, but it shouldn't actually ever change.